src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/node_modules/bootstrap/README.md

```diff
@@ -0,0 +1,239 @@
+<p align="center">
```

Comment:
Question, does this use Node modules? If so, that's going to require an additional build stage to the production dockerfile. We'll probably need to discuss this at the meeting tonight because we'll need to make sure everyone agrees with the method of keeping all the JS dependencies centralized.

Reply:
I believe that folder was created when I was attempting to learn how to customize bootstrap through sass files. My actual page does not implement any of it. Still learning how to do that though so it may in the future?